backend/program/content/mdblist.py

```diff
@@ -42,6 +42,8 @@ def run(self):
 
                 new_items = [item for item in items if item not in self.media_items]
                 container = self.updater.create_items(new_items)
+                for item in container:
+                    item.set_requested_by("Mdblist")

```

Comment:
item.set("requested_by", "mdblist") instead and remove the set_requested_by function

Reply:
Fixed